branch-diff.js

```diff
@@ -135,6 +135,11 @@ if (require.main === module) {
     , excludeLabels = []
     , options
 
+  if (argv.version || argv.v) {
+    console.log('v' + require('./package.json').version)
```

Comment:
template string and early-return (i.e. `return console.log`) then remove the braces around the `if`